modules/viewing-kueue-alerts-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,96 @@
+:_module-type: PROCEDURE
+
+[id="viewing-kueue-alerts-for-distributed-workloads_{context}"]
+= Viewing Kueue alerts for distributed workloads
+
+[role='_abstract']
+
+In {productname-short}, you can view Kueue alerts for your cluster.
+Each alert provides a link to a _runbook_.
+The runbook provides instructions on how to resolve the situation that triggered the alert.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You can access a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You can access a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You can access a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
+For information about projects and workbenches, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects[Working on data science projects].
+endif::[]
+ifdef::upstream[]
+* You can access a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
+For information about projects and workbenches, see link:{odhdocshome}/working-on-data-science-projects[Working on data science projects].
+endif::[]
+
+* You have logged in to {productname-long}.
+* Your data science project contains distributed workloads.
+
+.Procedure
+
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, click *Observe* -> *Alerting*.
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, click *Observe* -> *Alerting*.
+endif::[]
+
```

Comment:
Complete path should be mentioned here to reach "Alerting rules tab" like
Click Observe → Alerting and then click the Alerting rules tab

Reply:
The first part of the path is provided in the previous step -- see lines 41-46:
> In the OpenShift console, click *Observe* -> *Alerting*.

Reply:
I viewed this file and it doesn't show 

![Screenshot from 2024-09-12 16-48-34](https://github.com/user-attachments/assets/e257047c-96d4-4400-b057-f162f56e36e5)

Reply:
For some reason, it doesn't appear in preview since the statement is present, so am fine with this

Reply:
Conditionalised text doesn't show in generated previews in Git.

Reply:
Thank you @ChughShilpa !